contributing/contributing.mo.md

```diff
@@ -0,0 +1,42 @@
+## [MO] Contributing
+
+## Owner: [Yann Leflour](https://github.com/yleflour)
+
+## Prerequisites:
+
+- You have cloned [the repository](https://github.com/bamlab/dev-standards) locally
+- You have a markdown/text editor
+
+## Steps:
+
+### 1. Writting *(Variable time)*
+
+- For a Method of Operation please read [the following standard](/contributing/mo.s.md)
+- For a Standard please read [the following standard](/contributing/standard.s.md)
+- Create a branch and a pull request
+
+> **Check:** Your article is visible in the [pull requests list](https://github.com/bamlab/dev-standards/pulls)
+
+### 2. Review *(~3min)*
+
+- Assign 2 contributors for review
```

Comment:
Which one, how to choose ? we should use https://github.com/blog/2392-introducing-code-owners